codalab/worker_manager/slurm_batch_worker_manager.py

```diff
@@ -253,6 +253,8 @@ def setup_codalab_worker(self, worker_id):
             worker_network_prefix,
             # always set in the Slurm worker manager to ensure safe shutdown
             '--pass-down-termination',
+            '--exit-after-num-runs',
```

Comment:
instead of adding this here, can we do as below:

```
if self.args.exit_after_num_runs:
    command.extend(['--exit-after-num-runs', str(self.args.exit_after_num_runs)])
```

(same for the awsbatchworkermanager)